docs/Developers/Writing_Native_Bears.rst

```diff
@@ -110,8 +112,8 @@ You should now see an output like this on your command line:
     tutorial, else you will see a bunch of other outputs from other bears
     as well.
 
-The Bear class also supports ``warn`` and ``err`` to create ``WARNING`` and
-``ERROR`` messages respectively.
+The ``logging`` module also supports ``warning`` and ``error`` to
+create ``WARNING`` and ``ERROR`` messages respectively.
```

Comment:
IMO explaining about the logging module doesnt make sense.

Reply:
I think this line can't be left as it is, because the example code has been changed and doesn't use the Bear class' logging system. So it makes more sense to modify it accordingly.
But I'm waiting for feedback to know if I can rephrase it in another way.

Reply:
IMO you can remove this.

Reply:
IMO you can remove this.

Reply:
IMO you can remove this.

Reply:
sorry for multiple comment i was facing some net issues ;P